main_test.go

```diff
@@ -692,7 +694,7 @@ func TestInfoRequestHangs(t *testing.T) {
 	lambdaServerInternals := newMockLambdaServer(t, logsapiAddr, eventsChannel, l)
 
 	eventsChain := []MockEvent{
-		{Type: InvokeStandardInfo, APMServerBehavior: Hangs, ExecutionDuration: 1, Timeout: 500},
+		{Type: InvokeStandardInfo, APMServerBehavior: Hangs, ExecutionDuration: 1, Timeout: 5},
```

Comment:
[**For Reviewers**] I am not exactly sure about the purpose of this test. Based on the comment, it seems to test that extension times out in case APMServer hangs. Before this PR, the event process loop was terminated by sending of `RuntimeDone` event since the configured timeout in the test was `500` however, after the implementation of pushback in case no metadata is available `RuntimeDone` event is no longer processed. I have updated the `Timeout` to 5 so that it times out based on the deadline condition.

(for ref: [the logic to terminate the event processing loop](https://github.com/elastic/apm-aws-lambda/blob/995d42614a68d43bc531e15a36bcbee27606a9c4/app/run.go#L199-L213))